publishing-bot/manifests/deployment.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: publisher
+  namespace: publishing-bot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: publisher
+  template:
+    metadata:
+      labels:
+        name: publisher
+    spec:
```

Comment:
My question has basis as the ones before, where I'm not aware as how publishing bot works (I need to read about it), but should the spec be empty? Is it because it's a placeholder for the bot to fill?

Reply:
Makes sense :-)